src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,25 @@
+package calculator;
+
+import calculator.util.StringUtils;
+
+public class Calculator {
+
+    private static final String DELIMITER = " ";
+
```

Comment:
```suggestion
public class Calculator {
    private static final String DELIMITER = " ";

```
상수로 사용한 점 매우 좋습니다 👍 
그런데 상수는 class 바로 밑에 붙여서 선언해주세요! 

관련자료 : 자바 코딩 컨벤션
- 앞으로 미션 진행하면서 자바지기님께서 소개해주실테지만, 미리 알려드리겠습니다!
- https://google.github.io/styleguide/javaguide.html
- https://myeonguni.tistory.com/1596

Reply:
코딩컨벤션 관련 링크까지 친절한 코멘트 감사합니다. 읽어보겠습니다 !